### DISCUSSION TITLE
v0.4.1 | Configurable Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by ThijmenDam** [ThijmenDam/BarTranslate#16] [Announcements]
Hi folks, this release includes some nice usability improvements. 🚀

Configurable keyboard shortcuts to:

- Toggle the app.
- Swap languages.
- Select 'from' language.
- Select 'to' language.

All keyboard shortcuts (except for toggling the app) are now localized, i.e. they will only be registered when the BarTranslate window is open.

<hr /><em>This discussion was created from the release <a href='https://github.com/ThijmenDam/BarTranslate/releases/tag/0.4.0'>Configurable Keyboard Shortcuts | BarTranslate 0.4.0</a>.</em>

**Comment by peteruzek** [comment]
everytime i download the app, safari unzips it, it says the file is demagged and cannot be opened. i tried older versions too.

**Reply by ThijmenDam** [reply]
Hi @peteruzek, which chip do you have in your Mac? M1, M2 or Intel? And which architecture of the app did you download, arm64 or x64?

**Reply by peteruzek** [reply]
M1

**Reply by peteruzek** [reply]
i downloaded ARM, of course

**Reply by ThijmenDam** [reply]
Could you perhaps download this version and see if the issue is resolved?

https://drive.google.com/file/d/13BYHTJ7v0teos5Y0b8ZhxCDzHEdHK8LG/view?usp=share_link

If not, it might be worth it to check your security settings and confirm that the app is allowed to run. You could also try the x64 version and run it in Rosetta.

In the end its s hard for me to determine what might be wrong since I cannot replicate the issue. What I can do however is update all the app's dependencies, perhaps there is a known issue in Electron which might cause this problem. I'll let you know, though I do not have time to work on it before next week due to the Christmas holiday in the Netherlands. Another option is to build the app yourself, the instructions for doing so can be found in the readme.

**Reply by alanspina** [reply]
I have the same problem with my computer 

  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro18,1
  Model Number:	Z14V001EJLL/A
  Chip:	Apple M1 Pro
  Total Number of Cores:	10 (8 performance and 2 efficiency)
  Memory:	32 GB
  System Firmware Version:	8419.41.10
  OS Loader Version:	8419.41.10
 
 
 I tried with the drive version but I get the same error. I will try to build the app.

**Reply by ThijmenDam** [reply]
@alanspina thanks for building. If you still intend to build the app, could je perhaps let me know if that solved it for you?
